src/Url.roc

```diff
@@ -23,80 +23,80 @@ Url := Str
 
 ## Reserve the given number of bytes as extra capacity. This can avoid reallocation
 ## when calling multiple functions that increase the length of the URL.
+##
+## The following example reserves 50 bytes, then builds the url `https://example.com/stuff?caf%C3%A9=du%20Monde&email=hi%40example.com`;
 ## ```
 ## Url.fromStr "https://example.com"
-##     |> Url.reserve 50 # We're about to add 50 UTF-8 bytes to it
-##     |> Url.append "stuff"
-##     |> Url.appendParam "café" "du Monde"
-##     |> Url.appendParam "email" "hi@example.com"
-## # https://example.com/stuff?caf%C3%A9=du%20Monde&email=hi%40example.com
+## |> Url.reserve 50
+## |> Url.append "stuff"
+## |> Url.appendParam "café" "du Monde"
+## |> Url.appendParam "email" "hi@example.com"
 ## ```
-## The [Str.countUtf8Bytes] function can be helpful in finding out how many bytes to reserve.
+## The [Str.countUtf8Bytes](https://www.roc-lang.org/builtins/Str#countUtf8Bytes) function can be helpful in finding out how many bytes to reserve.
 ##
 ## There is no `Url.withCapacity` because it's better to reserve extra capacity
-## on a [Str] first, and then pass that string to [Url.fromStr]. This function will make use
+## on a [Str](https://www.roc-lang.org/builtins/Str) first, and then pass that string to [Url.fromStr]. This function will make use
 ## of the extra capacity.
 reserve : Url, Nat -> Url
 reserve = \@Url str, cap ->
     @Url (Str.reserve str cap)
 
 ## Create a [Url] without validating or [percent-encoding](https://en.wikipedia.org/wiki/Percent-encoding)
 ## anything.
+##
 ## ```
 ## Url.fromStr "https://example.com#stuff"
-## # https://example.com#stuff
+## ```
 ##
 ## URLs can be absolute, like `https://example.com`, or they can be relative, like `/blah`.
 ##
+## ```
 ## Url.fromStr "/this/is#relative"
-## # /this/is#relative
+## ```
 ##
 ## Since nothing is validated, this can return invalid URLs.
 ##
+## ```
 ## Url.fromStr "https://this is not a valid URL, not at all!"
-## # https://this is not a valid URL, not at all!
 ## ```
+##
 ## Naturally, passing invalid URLs to functions that need valid ones will tend to result in errors.
+##
 fromStr : Str -> Url
 fromStr = \str -> @Url str
 
-## Return a [Str] representation of this URL.
+## Return a [Str](https://www.roc-lang.org/builtins/Str) representation of this URL.
```

Comment:
Reverted all builtin links back to [Bool.true] style. Raised as an idea in Zulip. This will likely be resolved through a PR to `roc-lang/roc` cli docs gen, so another PR will be required over there to resolve.